ci-operator/config/opendatahub-io/odh-dashboard/opendatahub-io-odh-dashboard-main.yaml

```diff
@@ -39,6 +39,13 @@ tests:
     env:
       IMAGE_REPO: odh-dashboard
     workflow: opendatahub-io-ci-image-mirror
+- as: odh-dashboard-s390x-build
+  cron: 0 2 * * *
+  steps:
+    env:
+      IMAGE_REPO: odh-dashboard
+      RELEASE_VERSION: nightly
```

Comment:
Why `RELEASE_VERSION` is nightly? We are doing a cron job at night yes, but it's supposed to be exclusive for s390, why don't we call it something more descriptive?

Reply:
Sure, I'll change it in next commit.